cmd/proxy-server/main_test.go

```diff
@@ -109,3 +130,333 @@ func TestUpdateStorageSystems(t *testing.T) {
 		})
 	}
 }
+func TestVolumesHandler(t *testing.T) {
+	ctx := context.Background()
+	log := logrus.New().WithContext(ctx)
+	rdb = createRedisContainer(t)
+	sut := tenantsvc.NewTenantService(
+		tenantsvc.WithRedis(rdb),
+		tenantsvc.WithJWTSigningSecret("secret"),
+		tenantsvc.WithTokenManager(jwx.NewTokenManager(jwx.HS256)))
+
+	tests := map[string]func(t *testing.T, ctx context.Context, log *logrus.Entry){
```

Comment:
Can a test case for a volume having the `deleted` key be added?

Reply:
added